### PR TITLE
Debug-level logs, multi-viewer interface, and more tooltips

### PR DESCRIPTION
**Pull request by sirknightj**
*Issue #, if available:*

*Description of changes:*
* Master now can show multiple viewers
image

(pictured above: master and two viewers, with the mock video source: `open -a "Google Chrome" --args --use-fake-device-for-media-stream`. Make sure chrome is not running before launching otherwise the webcam might still show.)

* High-level overview 
  * When a new viewer connects, a new video element is added to the page, below the previous one.
  * When the peer connection moves to the FAILED state, the video element is removed from the page.
  * The viewer's `ClientId` is added below each video's media tracks.
  * When the master has no viewers connected, an empty video player will be shown.

* Add DEBUG log level
image

* log SDP offer and answer in DEBUG
* log generated ICE candidates in DEBUG
* log selected ice candidate in DEBUG
* log when the peer connection is successful or unsuccessful in DEBUG
* Add tooltip for ICE Candidate generation
image

image

-------
Fixed an issue with "Enable KVS WebRTC DQP Test and Metrics (Viewer only)" if the `ClientId` is empty, the display would generate a new one with each update.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -399,7 +399,7 @@ See [KVS WebRTC Access Control Documentation](https://docs.aws.amazon.com/kinesi
 #### Running WebRTC Test Page Locally
 The SDK and test page can be edited and run locally by following these instructions:
 
-NodeJS version 8+ is required.
+See the supported node versions in [ci.yml](.github/workflows/ci.yml).
```
Comment (niyatim23) [inline README.md:402]: Isn't it better to specify the versions directly in the readme?

File: examples/master.js [hunk 2]
```diff
@@ -199,13 +201,49 @@ async function startMaster(localView, remoteView, formValues, onStatsReport, onR
 
             // Poll for connection stats
             if (!master.peerConnectionStatsInterval) {
-                master.peerConnectionStatsInterval = setInterval(() => peerConnection.getStats().then(onStatsReport), 1000);
+                master.peerConnectionStatsInterval = setInterval(() => peerConnection.getStats().then(onStatsReport), 10000);
```
Comment (niyatim23) [inline examples/master.js:204]: Why are we increasing this?

File: README.md (hunk 1 above)
Comment (niyatim23) [inline README.md:402]:
```suggestion
NodeJS version 16+ is required.
```

File: examples/master.js (hunk 2 above)
Comment (sirknightj) [reply examples/master.js:204]: Previously it was a no-op firing every 1 second. Now, it's actually implemented, firing every 10 seconds.
Prints a lot of stuff.

**Review by niyatim23 — APPROVED** [review]
Nice

File: README.md (hunk 1 above)
Comment (sirknightj) [reply README.md]: Done

**Review by hassanctech — APPROVED** [review]

**Pull request closed without merging**